src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -736,13 +737,20 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
     protected function getContainerLoader(ContainerInterface $container)
     {
         $locator = new FileLocator($this);
-        $resolver = new LoaderResolver(array(
+        $loaders = array(
             new XmlFileLoader($container, $locator),
             new YamlFileLoader($container, $locator),
             new IniFileLoader($container, $locator),
             new PhpFileLoader($container, $locator),
             new ClosureLoader($container),
-        ));
+        );
+
+        // DirectoryLoader only appears in DI 2.7
+        if (class_exists('Symfony\\Component\\DependencyInjection\\Loader\\DirectoryLoader')) {
```

Comment:
I would prefer bumping the dependency and defining it all the time. Much better DX.